api-reference/commands/retry.md

```diff
@@ -0,0 +1,21 @@
+# retry
+
+Some flaky behaviour is expected or beyond an app's control. For those situations, it can be useful having a small controlled loop that will retry one or more commands a limited number of times.
```

Comment:
I think it would be useful to mention the flakiness examples: third party, animations not running in one machine but works on other

Reply:
Let's address this as follow up @Fishbowler !